### PR TITLE
add micromustache@3.0.4 w/ npm auto-update

### LINKED ISSUE #8598: [Request] Add Micromustache
**Library name:**
Micromustache
 
**Git repository url:**
https://github.com/userpixel/micromustache

**npm package url(optional):** 
https://www.npmjs.com/package/micromustache

**License(s):**
MIT

**Official homepage:**
https://github.com/userpixel/micromustache

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8598 
@x09326  please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/userpixel/micromustache
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/micromustache
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 35
   - Count of watchers: 2
   - Count of forks: 3
 * NPM download stats (optional):
   - Downloads in the last day: 210
   - Downloads in the last week: 715
   - Downloads in the last month: 1,059

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8598, cc @userpixel

### CODE REVIEW

File: ajax/libs/micromustache/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "template engine",
+    "microlibrary"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/userpixel/micromustache.git"
+  },
+  "author": "Alex Ewerlöf",
+  "license": "MIT",
+  "homepage": "",
```
Comment (PeterDaveHello) [inline ajax/libs/micromustache/package.json]: Empty homepage?

Comment (userpixel) [inline ajax/libs/micromustache/package.json]: Not sure what is means? It has a Github page.

Comment (kennynaoh) [inline ajax/libs/micromustache/package.json]: We usually add official website here not git repo.
Is there a website for this library?
If it isn't, I will remove the `"homepage"`.
Thanks!

Comment (userpixel) [inline ajax/libs/micromustache/package.json]: Not for this particular library. So I guess "homepage" will :hocho: 
Thanks!

Comment (kennynaoh) [inline ajax/libs/micromustache/package.json]: ok, thanks!

**Comment by kennynaoh** [comment]
@x09326 
I have removed the empty homepage.
Please help me review it again.
Thanks!

**Comment by userpixel** [comment]
I noticed that the minified version [doesn't show up](https://cdnjs.com/libraries/micromustache) on the CDN. Couldn't figure out how to fix that because the documentation says: 

> Notes that you should have a filename to point to the main file of a lib, if that file is not minified, please still use filename.min.js or filename.min.css structure naming, we'll do the minify job.

Is there anything I can do to add those minified files from the `/browser` directory without breaking Node.js compatibility?

**Comment by PeterDaveHello** [comment]
I have no problem from my side, @userpixel would you please give me a screenshot? Thanks.

**Comment by userpixel** [comment]
I just checked now. Seems to be OK. :+1:

**Pull request merged**